dev/README.md

```diff
@@ -1,70 +1,61 @@
 # Development Tomcat Environment
 
+It is possible to use Docker containers to test your code, but it is normally more convenient to simply run the repository webapp (**`alfresco.war`**) and Share webapp (**`share.war`**) in a tomcat instance. Options are also available to apply selected AMPs.
 
-It is possible to use Docker containers to test your code, but it is normally more convenient to simply run the
-repository webapp (`alfresco.war`) and Share webapp (`share.war`) in a tomcat instance. Options are also available to
-apply selected AMPs
-
-## Setting up your development environment
+## 1. Setting up your development environment
 Although it is possible to work on individual github projects, we recommend working on
 the `alfresco-community-repo`, `alfresco-community-share` and `acs-community-packaging`
 in a single Intellij IDEA project. They depend on each other and typically you 
 will want to make changes to all of them if you are changing the repository code.
 
-~~~
+~~~bash
 mkdir work
 cd work
 git clone git@github.com:Alfresco/alfresco-community-repo.git
 git clone git@github.com:Alfresco/alfresco-community-share.git
 git clone git@github.com:Alfresco/acs-community-packaging.git
 ~~~
 
-## Aliases
-There are a set of aliases to help with building. You may find them useful, as they will help you only build selected parts
-of the code base and will save you lots of typing.
+## 2. Aliases
+There are a set of aliases to help with building. You may find them useful, as they will help you only build selected parts of the code base and will save you lots of typing.
 
-Aliases ending in `D` provide Maven commands for building local Docker images. The AMPS environment variable will be of
-interest, if you wish to build AMPs included in the repo and share projects. 
+Aliases ending in **`D`** provide Maven commands for building local Docker images. The AMPS environment variable will be of interest, if you wish to build AMPs included in the repo and share projects. 
 
-The `aliases` file includes a more detailed description.
-~~~
+**You may also build **`alfresco-community-repository`**, **`alfresco-community-share`** and **`alfresco-community-packaging`** with Docker images at once with **`comD`**.**  
```

Comment:
I think there's no point in explaining this specific alias here, there's many aliases and they should be described appropriately in the `aliases` file already so that's the source of truth for this kind of information. We don't want to duplicate this so we'd have to maintain it in 2 separate places in case some of the aliases change.